src/posix.mak

```diff
@@ -486,12 +486,13 @@ clean:
 ######## Download and install the last dmd buildable without dmd
 
 ifneq (,$(AUTO_BOOTSTRAP))
+CURL_FLAGS:=-fsSL --retry 5 --retry-max-time 120 --connect-timeout 5 --speed-time 30 --speed-limit 1024
```

Comment:
A short explanation:
Retry max 5 times or max. for 120 seconds. The last three arguments are to prevent stalled connects or infinitesimal small download speed (the speed is measured in byes/second)